front/app/utils/permissions/rules/projectPermissions.ts

```diff
@@ -46,17 +46,32 @@ export function canModerateProject(
   user: IUser | undefined
 ) {
   const projectId = project.id;
-  const projectFolderId = project.attributes.folder_id;
+  const folderId = project.attributes.folder_id;
 
+  return canModerateProjectByIds({
+    projectId,
+    folderId,
+    user,
+  });
+}
```

Comment:
Since this `canModerateProject` utility was used in many places, and I didn't feel like rewriting all of them to use the ids instead of the full project object, I just added a second `canModerateProjectByIds` function which this one calls, and which can be also be used separately. Not perfect but saves a lot of refactoring